FirebaseVertexAI/Sample/GenerativeAITextSample/Screens/SummarizeScreen.swift

```diff
@@ -61,6 +65,10 @@ struct SummarizeScreen: View {
   }
 
   private func onSummarizeTapped() {
+    // MARK: Close Keyboard
+
```

Comment:
Nit: remove empty line

Reply:
Removed too. The empty line is added by the style script 🙈